packages/browser-pool/test/proxy-sugar.test.ts

```diff
@@ -0,0 +1,63 @@
+import { BrowserPool, PuppeteerPlugin, PlaywrightPlugin } from '@crawlee/browser-pool';
+import http from 'node:http';
+import type { AddressInfo } from 'node:net';
+import puppeteer from 'puppeteer';
+import playwright from 'playwright';
+import type { Server as ProxyChainServer } from 'proxy-chain';
+import { promisify } from 'node:util';
+import { createProxyServer } from '../../../test/browser-pool/browser-plugins/create-proxy-server';
```

Comment:
Isn't the `../` moot?

Reply:
Probably, I forget we have some aliases setup

Reply:
Nope, doesn't work... I think its because theres no mapping in the folder its in